### DISCUSSION TITLE
Feature Request: Ability to filter Item List to show any items that contain Better Notes

### DISCUSSION THREAD

**Discussion by DonnaCoxBaker** [windingwind/zotero-better-notes#1011] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS: Windows 11
- Zotero Version: 7.0.0-beta91 
- Plugin Version: 1.1.4-beta.87

### Describe the feature request

I have many notes in my Zotero database, but only certain ones are Better Notes--containing the links. These become my master notes, tying together many fragments of notes throughout my data. I would like to be able to narrow down the Item List to just the Better Notes I have worked on. I can add a tag, but I tend to forget. Could that be automated by the plugin?

### Describe the solution you'd like

It could either be a tag that is added when the first Better Notes activity is done within a note or could be a menu option that can find and display notes that have Better Notes content. I'd like to be able to pull the ones that have outbound links, without pulling the ones being linked to.

**Comment by windingwind** [comment, marked as answer]
Use `Saved Search`:

<img width="656" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/0ec55b5f-62d7-4c95-b290-a4f3b46dd949">

**Comment by DonnaCoxBaker** [comment]
That works! Thanks!
